src/components/transcript/index.tsx

```diff
@@ -183,28 +201,29 @@ const Transcript = ({ reviewData }: { reviewData: UserReviewData }) => {
     onSuccessCallback?: () => void,
     onNoEditsCallback?: () => void
   ) => {
+    // eslint-disable-next-line no-unused-vars
+    const { loc, title, date, body, media, transcript_by, ...restContent } =
+      updatedContent;
     // create an awaitable promise for mutation
 
-    const newImplData = {
-      directoryPath: updatedContent.loc?.trim() ?? "",
-      fileName: formatDataForMetadata(updatedContent.title),
-      url: transcriptData?.content.media,
-      date: updatedContent.date && dateFormatGeneral(updatedContent.date, true),
-      tags: formatDataForMetadata(updatedContent.tags),
-      speakers: formatDataForMetadata(updatedContent.speakers),
-      categories: formatDataForMetadata(updatedContent.categories),
-      transcribedText: updatedContent.body,
+    const newImplData: SaveToGHData = {
+      directoryPath: loc?.trim() ?? "",
+      fileName: formatDataForMetadata(title),
+      url: transcriptData.content.media,
+      date: date && dateFormatGeneral(date, true),
+      transcribedText: body,
       transcript_by: formatDataForMetadata(
         userSession?.user?.githubUsername ?? ""
       ),
       ghSourcePath,
       ghBranchUrl,
       reviewId: reviewData.id,
+      ...restContent,
     };
 
     const isPreviousHash = compareTranscriptBetweenSave(newImplData);
     if (isPreviousHash) {
-      onNoEditsCallback && onNoEditsCallback();
+      onNoEditsCallback?.();
```

Comment:
nit: This is hard to read. The verbose version is simple

Reply:
This is related to the other PR this is built on top of https://github.com/bitcointranscripts/transcription-review-front-end/pull/285